src/result_collection.jl

```diff
@@ -116,6 +120,18 @@ function collect_results!(filename, folder;
     else
         allfiles = joinpath.(Ref(folder), readdir(folder))
     end
+    
+    # Filter through entries
+    idx_del = Int[]
+    for i in eachindex(allfiles)
+        file = allfiles[i]
+        include_bool = any(match.(rinclude, file) .!== nothing)
+        exclude_bool = any(match.(rexclude, file) .!== nothing)
```

Comment:
Can we do this a bit more performantly so we don't allocate an array at each call? any can be used with iterators as well. `any(x == 2 for x in y)`